cmd/duration.go

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2022-2024 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package cmd
```

Comment:
Can this be useful for other Charon components? Then maybe move this to `app` or utils or something that can be shared.

Reply:
No problems, leave it here for now.